includes/discovery/sensors/fanspeed/dell.inc.php

```diff
@@ -16,12 +16,41 @@
 if (is_array($temp)) {
     foreach ($temp as $index => $entry) {
         $descr = $temp[$index]['coolingDeviceLocationName'];
-        $value = $temp[$index]['coolingDeviceReading'];
-        $lowlimit = $temp[$index]['coolingDeviceLowerCriticalThreshold'];
-        $low_warn_limit = $temp[$index]['coolingDeviceLowerNonCriticalThreshold'];
-        $warnlimit = $temp[$index]['coolingDeviceUpperNonCriticalThreshold'];
-        $limit = $temp[$index]['coolingDeviceUpperCriticalThreshold'];
+        if (isset($temp[$index]['coolingDeviceReading'])) {
```

Comment:
Well spotted with these.

To just tidy things up can you shorten these to:

`(isset($temp[$index]['coolingDeviceLowerCriticalThreshold'])) ? $value = $temp[$index]['coolingDeviceReading'] : $value = null;`